styles.css

```diff
@@ -650,37 +663,81 @@ body {
   }
 
   .item-nav {
-    background-image: url('images/Icon - Menu.png');
-    background-repeat: no-repeat;
-    padding: 8px;
-    position: absolute;
-    width: 30px;
-    height: 40px;
-    right: 24px;
-    left: 85%;
-    top: 30%;
+    display: none;
+    width: 100%;
+    height: 100%;
+    align-items: flex-start;
+    left: 10%;
+    top: 100%;
   }
 
-  .item-txt {
-    visibility: hidden;
-    left: 2%;
-    width: 0;
-    height: 0;
+  .toolbar {
+    flex-direction: column;
+    align-items: flex-start;
   }
 
-  .btn {
+  .toggle-button {
+    display: flex;
+    backdrop-filter: blur(4px);
+  }
+
+  .item-nav ul {
+    width: 100%;
+    flex-direction: column;
+    margin-top: 20%;
+    margin-left: 5%;
+  }
+
+  .item-nav li {
+    height: 32px;
+    font-size: 32px;
+    padding-bottom: 10%;
+  }
+
+  .item-nav a {
+    color: #fff;
+  }
+
+  .item-nav.active {
+    display: flex;
+    left: 0;
+    top: 0;
+    height: 100vh;
+    width: 100%;
+    backdrop-filter: blur(4px);
+  }
+
+  .item-nav .toggle-2 {
     position: absolute;
-    width: 24px;
-    height: 24px;
-    background-color: transparent;
-    top: 18px;
-    left: 85%;
+    color: #fff;
+    width: 25px;
+    height: 20px;
+    top: 1%;
+    left: 93%;
     border: none;
+    background-color: transparent;
   }
 
-  .btn:hover {
-    enable-background: white;
-    background-color: slateblue;
+  .section-1.change {
+    background: #7987ff;
+    width: 100%;
+    height: 100vh;
+  }
+
+  .dropdown {
+    display: flex;
+    flex-direction: column;
+    position: absolute;
+    width: 250px;
+    margin-top: 0.3rem;
+    background: #6070ff;
+    opacity: 0;
+  }
+
+  .show {
+    transform: translateY(0);
+    visibility: visible;
```

Comment:
- [ ] Kindly make sure that the mobile menu has a backdrop so that is matches the colour as shown in the attached loom video.